### ISSUE TITLE
Figure out way to link to R code.

### ISSUE THREAD

**Issue by ldecicco-USGS** [USGS-R/gsIntroR#1]

**Comment by ldecicco-USGS** [comment]
Could just use the R code from vignette (show students where that is). That's a little messy....I think beginners might be overwhelmed by commented out code, and code that isn't echo'ed. That may be where Jekyl shines in all honesty...it takes care of formatting that I'm putting into code.

The other obvious option is to create 1(?) R example code file that is a cleaned up version of all the examples.

**Comment by lawinslow** [comment]
Yeah, it's good to have clean examples. You don't want to have to explain "just ignore that stuff there". It dends to confuse them.

In my training, I leaned really heavily on examples I built while explaining how to do things. And people generally got good at copy-paste from the website. We didn't use the pretty R files that much.
